src/Microbiome.jl

```diff
@@ -32,8 +33,11 @@ import DataFrames: DataFrame
 import Clustering: Hclust, hclust
 import Base: getindex, setindex, length
 
+using Reexport
+@reexport using SpatialEcology
+import SpatialEcology.@forward_func
```

Comment:
I think you'd copied that from SpatialEcology already, with a dep you might as well just use that - this allows removing the utils.jl file

Reply:
Well, I copied it from a comment you made on gitter, but yeah, this makes sense :-)